subsys/shell/Kconfig.backends

```diff
@@ -135,6 +136,7 @@ if SHELL_BACKEND_RTT
 
 config SHELL_PROMPT_RTT
 	string "Displayed prompt name"
+	default "> " if SHELL_LIGHTWEIGHT
```

Comment:
Is it worth changing this and the ones below?

They do actually provide useful info
Plus they are probably not used in most boards - e.g. dummy is really just for tests I think

Reply:
Snuck this in here trying to get the prompt to look like something FAFT likes ... but I should make a different configuration option for that ... so dropping it